src/path/mod.rs

```diff
@@ -261,6 +261,14 @@ impl<'a> ValuePath<'a> for &'a str {
     }
 }
 
+impl<'a> ValuePath<'a> for &'a OwnedValuePath {
```

Comment:
What was the motivation for moving this here?

Reply:
I had an issue with the visibility of this trait implementation, but actually it should be in the PR for `keyvalue` filter - I'll double-check if I do need this and, if so, move to that PR. Reverting this change here.